source/agora/consensus/EnrollmentPool.d

```diff
@@ -295,34 +295,16 @@ private Enrollment createEnrollment(const ref Hash utxo_key,
 /// test for function of EnrollmentPool
 unittest
 {
-    import agora.common.Amount;
     import agora.consensus.data.ConsensusParams;
     import agora.consensus.data.Transaction;
-    import agora.consensus.Genesis;
     import std.algorithm;
-    import std.format;
-    import std.conv;
 
     scope storage = new TestUTXOSet;
-
-    auto gen_key_pair = WK.Keys.Genesis;
     KeyPair key_pair = KeyPair.random();
 
-    foreach (idx; 0 .. 8)
-    {
-        auto input = Input(hashFull(GenesisTransaction), idx.to!uint);
+    genesisSpendable().map!(txb => txb.refund(key_pair.address).sign(TxType.Freeze))
```

Comment:
We don't *have* to, but it's recommended.
In that case, the reason it wasn't done was because the test uses the same key for all enrollments. It's something I wanted to fix, but decided to do it in a separate commit as that is not blocking me on #821 right now.